File: plugin/Template/src/main/java/alt.archimedes5000.plugins/Template.java
package alt.archimedes5000.plugins;

import alt.archimedes5000.plugins.utils.*;
import com.aliucord.utils.*;
import com.aliucord.annotations.AliucordPlugin;
import com.aliucord.entities.Plugin;
import android.content.Context;
import com.aliucord.patcher.*;

@AliucordPlugin(true)
class Template extends Plugin{
	@Override void start(Context pluginContext){
	};
	@Override void stop(Context pluginContext){
		patcher.unpatchAll();
	};
};

File: plugin/Template/src/main/kotlin/alt.archimedes5000.plugins/Template.kt
package alt.archimedes5000.plugins;

import alt.archimedes5000.plugins.utils.*;
import com.aliucord.utils.*;
import com.aliucord.annotations.AliucordPlugin;
import com.aliucord.entities.Plugin;
import android.content.Context;
import com.aliucord.patcher.*;

@AliucordPlugin(requiresRestart = true)
class Template: Plugin(){
	override fun start(pluginContext: Context){
	};
	override fun stop(pluginContext: Context){
		patcher.unpatchAll();
	};
};